### DISCUSSION TITLE
How add new controller and view file under created module

### DISCUSSION THREAD

**Discussion by ChamilaShantha** [nasirkhan/laravel-starter#602] [General]
I want to know what happens after I create a module After that, i want to add my old project controller and view file to created module? I  already add controller code to existing backend controller and create new view file to it but it not display in existing layout. it display out side of the layout.

**Comment by nasirkhan** [comment]
Well, you may just copy-paste the old project controllers, views, and other files to this new project. Check the route, and controller namespaces and follow the error messages when you paste the old code. 

Laravel Starter is a Laravel application and does not have any conflict with the official documentation.

**Reply by ChamilaShantha** [reply]
i copied and pasted my old project controllers, views, and other files to this new project. but it not display inside the layout.
